tests/flake8_debug_test.py

```diff
@@ -77,3 +78,19 @@ def test_present_multiple_breakpointhooks():
     ) == tuple(
         _out(line, column=5, err=BreakpointHookError()) for line in (3, 4)
     )
+
+
+def test_present_set_trace():
+    assert _plugin_results(
+        'def f():\n    import pdb\n    pdb.set_trace()'
+    ) == (
```

Comment:
probably we need to add some assert messages for all test funcs, but this is out of this PR